unikernel.ml

```diff
@@ -1045,8 +1045,8 @@ struct
           ~title:"Albatross Error" ~api_meth:false `Internal_server_error reqd
           ()
     | Ok (unikernel_name, unikernel) -> (
-        Builder_web.send_request http_client
-          ("/hash?sha256=" ^ Ohex.encode unikernel.digest)
+        Utils.send_http_request http_client ~base_url:Builder_web.base_url
+          ~path:("/hash?sha256=" ^ Ohex.encode unikernel.digest)
```

Comment:
same as above, makes the PR rather hard to review. Maybe next time split this into a separate PR?

Reply:
same as above, please separate such refactorings into a separate PR in the future.